helpers/models/common.py

```diff
@@ -869,7 +869,9 @@ def uses_shared_modules(self):
         # shared modules may be when ControlNet reuses base model layers, eg. HiDream.
         return False
 
-    def get_trained_component(self, base_model: bool = False):
+    def get_trained_component(self, base_model: bool = False, unwrap_model: bool = False):
+        if not unwrap_model:
+            return self.model
```

Comment:
i really don't want to change this default behaviour, that's quite a large change. why are you adding it?